### DISCUSSION TITLE
Specify TTL for tokens outside of global yml config

### DISCUSSION THREAD

**Discussion by kotyara85** [ory/hydra#3099] [Q&A] [answered]
Hello, is it possible to specify a custom exp date outside of global config on login or consent?
We need different TTLs for different type of tokens (up to one year) for service jwt
Thanks

**Comment by vinckr** [comment, marked as answer]
You can set different TTLs for 
[ID Token](https://www.ory.sh/docs/hydra/guides/token-expiration#id-token-expiration)
&
[Auth Code](https://www.ory.sh/docs/hydra/guides/token-expiration#auth-code-expiration)

Was that what you were looking for?

**Reply by kotyara85** [reply]
No, those look like a global config so all tokens working the system will have the same ttl. 
I need different ttl per different group of token. 
thanks

**Comment by kotyara85** [comment]
[3101](https://github.com/ory/hydra/discussions/3101)
